client/app/domain/tasks/TASKS.html

```diff
@@ -1,71 +1,35 @@
-                        <td data-ng-bind="::task.comment || '-'"></td>
-                        <td class="text-center">
-                            <span class="label"
-                                  data-ng-class="{
-                                    'label-info': task.status === 'TODO' || task.status === 'DOING',
-                                    'label-danger': task.status === 'ERROR',
-                                    'label-default': task.status === 'CANCELLED',
-                                    'label-success': task.status === 'DONE'
-                                  }"
-                                  data-ng-bind="tr('domain_tab_TASKS_status_' +  task.status)">
-                            </span>
-                        </td>
-                        <td>
-                            <button class="btn btn-icon" type="button" title="{{i18n.wizard_modify}}"
-                                    data-ng-if="task.canCancel || task.canRelauch || task.canAccelerate"
-                                    data-ng-click="setAction('../domain-operation/update/domain-operation-update', task)">
-                                <span class="oui-icon oui-icon-pen_line oui-icon_small" aria-hidden="true"></span>
-                            </button>
-                        </td>
-                    </tr>
-                </tbody>
-            </table>
-        </div>
-    </div>
-
-    <div class="clearfix" data-ng-if="ctrlDomainTasks.tasks">
-        <div data-pagination-front
-             data-items="ctrlDomainTasks.tasks"
-             data-paginated-items="ctrlDomainTasks.tasksDetails"
-             data-current-page="currentPage"
-             data-nb-pages="nbPages"
-             data-items-per-page="itemsPerPage"
-             data-transform-item="ctrlDomainTasks.transformItem(item)"
-             data-on-transform-item-get-promise="getPromise(promise)"
-             data-on-transform-item-notify="ctrlDomainTasks.onTransformItemNotify(item)"
-             data-on-transform-item-done="ctrlDomainTasks.onTransformItemDone(items)"
-             data-page-placeholder="{{tr('pagination_page', [currentPage, nbPages])}}"
-             data-item-per-page-placeholder="{{tr('pagination_display')}}"
-             data-ng-show="!ctrlDomainTasks.loading && ctrlDomainTasks.tasksDetails.length > 0"></div>
-    </div>
+    <oui-datagrid data-rows="ctrlDomainTasks.tasks"
+        data-row-loader="ctrlDomainTasks.transformItem($row)">
+        <oui-column data-property="function"
+            data-title="tr('domain_tab_TASKS_function')"></oui-column>
+        <oui-column data-property="todoDate"
+            data-title="tr('domain_tab_TASKS_function')">
+            {{ $row.todoDate | date:'mediumDate' }}
+        </oui-column>
+        <oui-column data-property="comment"
+            data-title="tr('domain_tab_TASKS_comment')">
+            {{ $row.comment || '-' }}
+        </oui-column>
+        <oui-column property="status"
```

Comment:
It's strange to use both property= and data-property=

Reply:
Yes!